initializer/initializer.go

```diff
@@ -111,6 +112,73 @@ func (i *Initializer) Init(ctx context.Context, args []string) error {
 	return nil
 }
 
+func (i *Initializer) createGitHook() {
+    b := []byte(`#!/bin/bash
+# Upload current directory to skynet if branch is dg-pages
+
+echo "Running update hook"
+
+remote=$1
+remote_url="$2"
+
+
+echo "Remote name is $remote"
+echo "Remote URL is $remote_url"
+
+while read local_ref remote_ref
+do
+    echo "Local branch $local_ref"
+
+    if [ "$local_ref" == "refs/heads/dg-pages" ]
+    then
+        # Push to skynet
+        echo "Publishing your pages to Sia (Skynet)"
+        mkdir _pages
+        rsync -am --include='*.css' --include='*.js' --include='*.html' --include='*/' --exclude='*' ./ _pages
+        touch _pages/_e2kdie_
+        skynet upload _pages
+        rm -rf _pages
+    fi
+done
+
+exit 0
+    `)
+
+    c:= []byte(`package main
```

Comment:
This part is probably best done as part of the CLI of dgit itself

Reply:
Okk. Is there a manual for the commands? dgit --help doesn't bring up a lot of options

Reply:
Sorry for awol, I've had a hectic two weeks. I moved the git hook creation block to init.go